backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -137,24 +83,42 @@ private MimeMessage createEmail(String subject, String bodyText) throws Messagin
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
     email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
-    email.setSubject(subject);
+    email.setSubject(subjectText);
     email.setText(bodyText);
     return email;
   }
 
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String recipientName, String recipientEmail)
+      throws MessagingException, IOException {
     MimeMessage email =
         createEmail(
-            "Ad-Lib Meeting Found",
-            " Hey "
-                + recipientName
-                + " Please Join your Ad-Lib meeting via the link below : \n"
-                + " http://meet.google.com/new");
+            recipientEmail,
+            "Ad-lib: We found you a match!",
+            String.format(
+                " We found you a match with %s "
+                    + " Check your calendar for your meeting event,"
+                    + " and feel free to join the Meet call now!\n",
+                recipientName));
     Message messageWithEmail = createMessageWithEmail(email);
     service.users().messages().send("me", messageWithEmail).execute();
```

Comment:
Change this `me` to `AUTH_USERNAME` since we've already created a variable

Reply:
Ack